crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -903,40 +903,6 @@ impl IndexerStore for PgIndexerStore {
         }).context(&format!("Failed reading transaction digests with kind {kind} and start_sequence {start_sequence:?} and limit {limit}"))
     }
 
-    async fn get_transaction_page_by_mutated_object(
```

Comment:
this is no longer needed b/c we now have a dedicated indexing table and this search was very slow.